cmd/operator-sdk/new/cmd.go

```diff
@@ -343,6 +344,7 @@ func doAnsibleScaffold() error {
 	return nil
 }
 
+//deprecated: it will be replaced by the plugin
 func doHelmScaffold() error {
```

Comment:
It will be removed 100% in the next step. Follow up PR where we will make the command call the plugin.

Reply:
See the follow up: https://github.com/operator-framework/operator-sdk/pull/3196/commits/f71c3ea59e4425737a883daaa505c559cf0cdb0f